src/firebolt_ingest/table_model.py

```diff
@@ -92,7 +95,7 @@ def alias_validator(cls, values: dict):
 
 FILE_METADATA_COLUMNS: List[Column] = [
     Column(name="source_file_name", type="TEXT"),
-    Column(name="source_file_timestamp", type="TIMESTAMP"),
+    Column(name="source_file_timestamp", type="TIMESTAMPNTZ"),
```

Comment:
There is an issue with this constant - https://firebolt-analytics.slack.com/archives/C027G8C7XLY/p1697020448394419. So when you select FROM information_schema.columns you will always see source_file_timestamp as TIMESTAMP even if it was TIMESTAMPNTZ. so then the code in insert_full_overwrite will not add source_file_timestamp to the insert query
```

        if set((c.name, c.type) for c in FILE_METADATA_COLUMNS).issubset(
            internal_table_columns
        ):
            column_names.append("source_file_name")
            column_names.append("source_file_timestamp")
```

Reply:
Adding another comment from the thread https://firebolt-analytics.slack.com/archives/C027G8C7XLY/p1697025409945839?thread_ts=1697020448.394419&cid=C027G8C7XLY 

_In all outputs (except the actual sql ddl string) you should never see the word "timestampNTZ" - instead you should always see "timestamp". Why? Because TimestampNtz is replacing the legacy timestamp and therefore we display them equally._